lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py

```diff
@@ -26,9 +26,20 @@
     name:
         description:
             - Limit results to a specific virtual machine scale set
+        required: false
+        default: null
     resource_group:
         description:
             - The resource group to search for the desired virtual machine scale set
+        required: false
+    format:
+        description:
+            - What data should be returned?
```

Comment:
Would be good to get a more detailed description of each option, e.g `raw` is the raw values returned by the SDK while `curated` is a more human friendly output with a flatter result. Does it change the return values format and so on.

Reply:
Also information saying in pre 2.5 this would always return the raw value